contracts/OffsetHelper.sol

```diff
@@ -136,11 +166,21 @@ contract OffsetHelper is OffsetHelperStorage {
             "Can't swap this token"
         );
```

Comment:
We can remove this check now that it is executed in `calculateNeededTokenAmount`